website/docs/reference/glossary.md

```diff
@@ -146,6 +146,12 @@ See [Operation modes](../configuration/operation-modes.md) for more details.
 An open source relational database.
 FerretDB uses PostgreSQL as a database engine.
 
+#### Pushdown
```

Comment:
The issue is about clarifying terminology. We don't have a term "pushdown"

Reply:
The issue mentioned glossary in TODO list, what changes in glossary should be applied?